force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +66,18 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
-        
+        List<Boolean> wordsToCheck = new List<Boolean>();
+        String stringToCheck = input;
         // Check if the string starts with 'Sales'
-        
+        Boolean firstCharactersMatch = (stringToCheck.left(5) == 'Sales');
```

Comment:
There are actually some shortcut methods, but I think this is fine in my book. See `String.startsWith` and `String.endsWith`, but those are unique to Java/Apex so thinking more abstractly is not a bad thing.